data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,405 @@
+					goto end
+				label flee
+				`	You drop the repair equipment and run back into your ship, smashing the button that closes the lock. You get a last look of one of the aliens: blue and yellow tones are mixing on their skin, but they are not taking any action.`
+				label end
+				`	As you fly away you see the engines of the ship slowly booting back up. Apparently they figured out how to use the repair equipment. Instead of leaving the system, however, the ship quietly disappears into the nearest gas giant.`
+				`	Back on the bridge your eyes fall on the scanner log that is still open on your screen. Notably this ship had no hyperdrive or anything similar installed; strange, considering there is no inhabited planet in this system. Maybe you should talk to the Quarg and see if they can help.`
+				action
+					set "incipias: helped the derelict"
+		personality
+			confusion 50
+			derelict staying uninterested
+	on enter "Hui'uc"
+		conversation
+			`On entering the system you find an alien spaceship disabled and badly damaged. There do not appear to be any other ships or inhabited planets in the system.`
+
+mission "Incipias: Help The Stranded 1"
+	landing
+	source
+		government "Quarg (Incipias)"
+	destination "Pon'tes"
+	to offer
+		has "incipias: helped the derelict"
+	on offer
+		conversation
+			`Maybe the Quarg could help you with the stranded ship.`
+			choice
+				`	(Tell them about it.)`
+				`	(Not right now.)`
+					defer
+			`	Right after you arrive on the planet, you search for a Quarg who speaks your language so that you can tell them about the stranded ship you found. One of the Quarg seems to understand your intention and gestures for you to wait. It returns a few hours later accompanied by the same Quarg you talked to before.`
+			`	"Human, you seem to be in quite a haste. What could be such a pressing matter?" it asks.`
+			choice
+				`	(Explain the situation.)`
+					goto explanation
+				`	"I need advice regarding the species in this region."`
+			`	It looks at you curiously. "So you have held council with the young Incipias? This one is nonplussed that you ably formed correspondence. It took a prolonged period of time even for us to decipher their way of dissemination."`
+			`	You quickly explain what happened. From what you can tell, the Quarg seems relieved that you were not able to talk to them.`
+			label explanation
+			`	"You ran into a most peculiar situation. Surpassing the borders of their system is not yet a capability possessed by the Incipias." It tilts its head for a moment, considering what you just said. "It is our desire to help them, but it is yet premature for us to reveal ourselves to their species, and certainly premature for them to be interacting with independent interlopers such as yourself. I fear that we are impotent in this unfortunate turn of events."`
+			choice
+				`	"Then let me help them."`
```

Comment:
```suggestion
				`	"I could help them."`
```
This early in the conversation it'd be better to suggest you can help them, and then later on be more pressing about letting you do so.